NBitcoin/TransactionBuilder.cs

```diff
@@ -1153,7 +1153,9 @@ public TransactionBuilder Shuffle()
 
 		private void DoShuffle()
 		{
-			if (ShuffleRandom != null)
+			TransactionBuildingContext ctx = new TransactionBuildingContext(this);

```

Comment:
remove this, this is uneeded.

Reply:
@NicolasDorier how can he check ctx.CanShuffleOutputs then? should pass ctx as param?